drivers/gpu/drm/tests/drm_framebuffer_test.c

```diff
@@ -337,6 +339,7 @@ static int drm_framebuffer_test_init(struct kunit *test)
 	mock = kunit_kzalloc(test, sizeof(*mock), GFP_KERNEL);
 	KUNIT_ASSERT_NOT_ERR_OR_NULL(test, mock);
 
+	mutex_init(&mock->mode_config.fb_lock);
```

Comment:
We need to destroy this mutex at some point or use `drmm_mutex`

Reply:
Thank you!
I Just added an exit function to the suite and put a `mutex_destroy` on it in that last force-push.